### ISSUE TITLE
Failed to create MidiOutput on macOS

### ISSUE THREAD

**Issue by RobertBendun** [RobertBendun/harmonia#11]
Some unknown problem in MidiOutput creation on macOS:

```
2024-04-26T16:35:03.396260Z  INFO harmonia: starting up version 0.1.1 (5e9b3f0 2024-04-26 08:48)
2024-04-26T16:35:03.411748Z ERROR linky_groups::net: failed to open multicast socket for interface 192.168.1.101: Address already in use (os error 48)
2024-04-26T16:35:03.411869Z  WARN harmonia: trying to recollect previous sources: opening state file: No such file or directory (os error 2)
2024-04-26T16:35:03.411917Z  INFO harmonia: link active
2024-04-26T16:35:03.411920Z  INFO linky_groups::net: Started linky_groups
2024-04-26T16:35:03.412342Z  INFO harmonia: Listening on http://127.0.0.1:8080/
thread 'tokio-runtime-worker' panicked at src/harmonia.rs:777:43:
called `Result::unwrap()` on an `Err` value: InitError
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'tokio-runtime-worker' panicked at src/harmonia.rs:777:43:
called `Result::unwrap()` on an `Err` value: InitError
thread 'tokio-runtime-worker' panicked at src/harmonia.rs:777:43:
called `Result::unwrap()` on an `Err` value: InitError
```

Couldn't reproduce for now, let's see if others have the same issue.

To maybe reproduce just try launching Harmonia on newer macOS.

**Comment by SamPanDonte** [comment]
I reproduced this error on a MacBook M1 (2020). For some reason, it sometimes worked correctly and I could use the frontend interface, but sometimes it did not. Os version: 14.5 (23F79).

**Comment by RobertBendun** [comment]
So it seems to be an dependency issue, solved in 950232087a4c32cc7b52bd75ee3fb1a60e305d32, but I would appreciate if you could confirm it on your machine. For now I'm closing the issue since seems solved
